shell.nix

```diff
@@ -0,0 +1,29 @@
+let
```

Comment:
@ari-becker one important thing to remeber is that the infra image is working with older version